app.js

```diff
@@ -2,13 +2,14 @@ const express = require("express");
 const app = express();
 app.use(express.json());
 
-const { getTopics } = require("./controllers/data.controller");
+const { getTopics, getArticles } = require("./controllers/data.controller");
 const {
   handlePSQL400s,
   handleCustomErrors,
 } = require("./controllers/error-handling.controller");
 
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:articles_id", getArticles);
 
 app.use(handlePSQL400s);
 app.use(handleCustomErrors);
```

Comment:
I'd also add an error handler to catch all other errors, (500 Server Error Handler)